_posts/2019-02-02-digitalocean-one-click.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Announcing DigitalOcean's OpenFaaS One-click"
+description: Earlier this week saw the release of the first OpenFaaS One-click application on DigitalOcean.  Richard Gee guides you through just how simple it is to create your first OpenFaaS function on DigitalOcean.
+date: 2019-02-02
+image: /images/digitalocean-one-click/balance-macro-ocean.jpg
```

Comment:
Please see my guidelines for resizing the lead image to ~= 280KB in the README.md

Reply:
I like this theme. Can we find an image that has a sea-blue with some waves to show the ocean theme? Or something a bit more vibrant?

https://www.pexels.com/photo/photography-of-barrel-wave-1298684/
https://www.pexels.com/photo/ocean-wave-1656579
https://www.pexels.com/photo/beach-calm-clouds-horizon-459522/
https://www.pexels.com/photo/beach-beautiful-blue-sky-blue-water-459252/

Reply:
I was trying to be more subtle than a picture of an ocean.  Quite liked the idea of a stack in the foreground.